tabs/pid_tuning.js

```diff
@@ -17,15 +17,20 @@ TABS.pid_tuning.initialize = function (callback) {
         if (semver.gte(CONFIG.apiVersion, CONFIGURATOR.pidControllerChangeMinApiVersion)) {
             return MSP.promise(MSP_codes.MSP_PID_CONTROLLER);
         }
-        return true;
     }).then(function() {
         return MSP.promise(MSP_codes.MSP_PIDNAMES)
     }).then(function() {
         return MSP.promise(MSP_codes.MSP_PID);
+    }).then(function () {
+        if (semver.gte(CONFIG.flightControllerVersion, "2.9.0") && semver.lt(CONFIG.flightControllerVersion, "3.0.0")) {
+          return MSP.promise(MSP_codes.MSP_SPECIAL_PARAMETERS);
+        }
     }).then(function() {
-        return MSP.promise(MSP_codes.MSP_RC_TUNING);
+        if (semver.gte(CONFIG.flightControllerVersion, "2.8.2")) {
+          return MSP.promise(MSP_codes.MSP_PID_ADVANCED);
```

Comment:
Why the reordering of `MSP_codes.MSP_PID_ADVANCED` and `MSP_codes.MSP_RC_TUNING`?